### DISCUSSION TITLE
Guitar Toolkit Scales question

### DISCUSSION THREAD

**Discussion by atuline** [soundshed/soundshed-app#73] [General]
I like this feature in soundshed, and saw that you have the Pentatonic Minor scale in there, but no Major Pentatonic scale.

And yes, I know that Em = G and Am = C and so on, but it's a pain to try and convert on the fly. And it IS a popular scale:

https://appliedguitartheory.com/lessons/major-pentatonic-scale/

Sure would be nice to see that added, however, with a last release date of 2022, I don't really see this happening.

**Comment by webprofusion-chrisc** [comment]
Sure thing, the guitar toolkit is loaded from https://scalex.soundshed.com and the source for that is at https://github.com/webprofusion/scalex. The scales are defined here: https://github.com/webprofusion/scalex/blob/f1d285b74476e0bf9630297493955f805b0c8e3d/src/Core/General/Scales.cs#L167 - it would be a fairly easy PR if you want to try.

Background: I wrote that app in about 2001 and have ported it a few times but it's much the same code, which was originally based on something I wrote in pascal in 1993..
